docs/dev/setup/jenkins-gitlab.rst

```diff
@@ -733,6 +733,95 @@ Configure your master node like this  (adjust the number of executors, if needed
 
       Jenkins local node
 
+Alternative local build agents setup using docker
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+An alternative way of adding a build agent that will use docker (similar to the remote agents below) but running
+locally, can be done using the jenkins/ssh-agent docker image (https://hub.docker.com/r/jenkins/ssh-agent).
+
+Prerequisites:
+
+1. Make sure to have Docker installed: https://docs.docker.com/engine/install/ubuntu/
+
+Agent setup:
+
+1. Create a new SSH key using ``ssh-keygen`` (if a passphrase is added, store it for later)
+
+2. Copy the public key content (e.g. in ~/.ssh/id_rsa.pub)
+
+3. Run ``docker run -d --name jenkins_agent -v /var/run/docker.sock:/var/run/docker.sock jenkins/ssh-agent:latest "<copied_public_key>"``
```

Comment:
This command appears to be too long and doesn't render correctly:
![grafik](https://user-images.githubusercontent.com/72132281/104104749-4e821d00-52aa-11eb-98e3-32af7a09fb74.png)